examples/decoding/plot_haxby_grid_search.py

```diff
@@ -38,6 +38,10 @@
 import numpy as np
 haxby_dataset = datasets.fetch_haxby_simple()
 
+# print basic information on the dataset
+print('Mask nifti images are located at: ' % haxby_dataset.mask[0])
+print('Functional nifti images are located at: ' % haxby_dataset.func[0])
```

Comment:
For, `haxby_dataset.func` is a string. Can you make sure all the examples run fine to get rid for other mistakes like this ?